### ISSUE TITLE
Error with curl command in install.sh script: "Unknown option: ignore-missing" on Amazon Linux 2

### ISSUE THREAD

**Issue by K-Oxon** [rilldata/rill#2141]
I am trying to install Rill on Amazon Linux 2 using the following command:

```bash
curl -s https://cdn.rilldata.com/install.sh | bash
```

The installation script seemed to run without any issues until it reached the verification step:

```bash
Verifying the SHA256 checksum of the downloaded binary:
Unknown option: ignore-missing
Type shasum -h for help
```

It seems that the shasum command used in the script does not support the --ignore-missing option. Is there a workaround for this issue on Amazon Linux 2?

Thank you for your help.

**Comment by mindspank** [comment]
Late to the party but the workaround could be to switch shasum package, `yum install -y perl-Digest-SHA`.

**Comment by lgonzalezsa** [comment]
I am in OpenSuse 15.5 and getting the same issue with shasum package (perl-Digest-SHA).

```bash
gonluisr@server:~/analytics> rill upgrade
A new version of rill is available (run `rill upgrade`): 0.42.3 → v0.52.3

Downloading binary: https://cdn.rilldata.com/rill/v0.52.3/rill_linux_amd64.zip
######################################################################## 100.0%

Downloading checksum: https://cdn.rilldata.com/rill/v0.52.3/checksums.txt
######################################################################## 100.0%

Verifying the SHA256 checksum of the downloaded binary:
Unknown option: ignore-missing
Type shasum -h for help
Error: exit status 1
```
